src/pages/workspaces/workspace/Dashboard/Dashboard.test.ts

```diff
@@ -189,7 +206,10 @@ describe('WorkspaceNotifications', () => {
 });
 
 describe('BucketLocation', () => {
-  const workspace = { workspace: { namespace: 'test', name: 'test', cloudPlatform: 'Gcp' }, workspaceInitialized: true };
+  const workspace = {
```

Comment:
It would be good to add some type safety to mock data in these tests.

```suggestion
  const workspace: WorkspaceWrapper = {
```